### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ankit-akash**
### Describe your change:

* [x] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x]  All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: knapsack/assignment_problem.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# A Dynamic Programming based Python Program for 0-1 Knapsack problem
```
Comment (realstealthninja) [inline knapsack/assignment_problem.py:1]: Add a link please

File: knapsack/assignment_problem.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+def knapSack(W, wt, val, n):
```
Comment (realstealthninja) [inline knapsack/assignment_problem.py:4]: Requires doctests and a docstring

File: knapsack/assignment_problem.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                K[i][w] = K[i - 1][w]
+
+    return K[n][W]
+
+
+val = [60, 100, 120]
+wt = [10, 20, 30]
+W = 50
+n = len(val)
+print(knapSack(W, wt, val, n))
```
Comment (realstealthninja) [inline knapsack/assignment_problem.py:22]: Either put it inside a a check for main or delete it entirely

**Review by realstealthninja — CHANGES_REQUESTED** [review]

**Comment by ZeroDayOwl** [comment]
@ankit-akash maybe change your PR title to whatever you are proposing to do.

File: knapsack/assignment_problem.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    K = [[0 for x in range(W + 1)] for x in range(n + 1)]
+    for i in range(n + 1):
+        for w in range(W + 1):
+            if i == 0 or w == 0:
+                K[i][w] = 0
+            elif wt[i - 1] <= w:
+                K[i][w] = max(val[i - 1] + K[i - 1][w - wt[i - 1]], K[i - 1][w])
+            else:
+                K[i][w] = K[i - 1][w]
+
```
Comment (ZeroDayOwl) [inline knapsack/assignment_problem.py:14]: Perhaps add comments explaining how the function works here.

**Review by ZeroDayOwl — CHANGES_REQUESTED** [review]

**Comment by cclauss** [comment]
Duplicate https://github.com/TheAlgorithms/Python/search?q=knapsack&type=code

**Pull request closed without merging**